packages/flutter/test/material/date_picker_test.dart

```diff
@@ -779,401 +752,13 @@ void main() {
-        of: find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_BorderContainer'),
-        matching: find.byWidgetPredicate((Widget w) => w is CustomPaint),
-      ));
-      final dynamic/*_InputBorderPainter*/ inputBorderPainter = customPaint.foregroundPainter;
-      final dynamic/*_InputBorderTween*/ inputBorderTween = inputBorderPainter.border;
-      final Animation<double> animation = inputBorderPainter.borderAnimation as Animation<double>;
-      final InputBorder actualBorder = inputBorderTween.evaluate(animation) as InputBorder;
-      final Color containerColor = inputBorderPainter.blendedColor as Color;
-
-      // Border should match
-      expect(actualBorder, equals(border));
-
-      // It shouldn't be filled, so the color should be transparent
-      expect(containerColor, equals(Colors.transparent));
-    });
-  });
-
-  group('CalendarDatePicker', () {
-    // Tests for the standalone CalendarDatePicker class
-    testWidgets('Updates to initialDate parameter is reflected in the state', (WidgetTester tester) async {
-      final Key pickerKey = UniqueKey();
-      final DateTime initialDate = DateTime(2020, 1, 21);
-      final DateTime updatedDate = DateTime(1976, 2, 23);
-      const Color selectedColor = Color(0xff2196f3); // default primary color
-
-      await tester.pumpWidget(MaterialApp(
-        home: Material(
-          child: CalendarDatePicker(
-            key: pickerKey,
-            initialDate: initialDate,
-            firstDate: DateTime(1970, 1, 1),
-            lastDate: DateTime(2099, 31, 12),
-            onDateChanged: (DateTime value) {},
-          ),
-        ),
-      ));
-      await tester.pumpAndSettle();
-
-      // Month should show as January 2020
-      expect(find.text('January 2020'), findsOneWidget);
-      // Selected date should be painted with a colored circle
-      expect(
-        Material.of(tester.element(find.text('21'))),
-        paints..circle(color: selectedColor, style: PaintingStyle.fill)
-      );
-
-      // Change to the updated initialDate
-      await tester.pumpWidget(MaterialApp(
-        home: Material(
-          child: CalendarDatePicker(
-            key: pickerKey,
-            initialDate: updatedDate,
-            firstDate: DateTime(1970, 1, 1),
-            lastDate: DateTime(2099, 31, 12),
-            onDateChanged: (DateTime value) {},
-          ),
-        ),
-      ));
-      // Wait for the page scroll animation to finish
-      await tester.pumpAndSettle(const Duration(milliseconds: 200));
-
-      // Month should show as February 1976
-      expect(find.text('January 2020'), findsNothing);
-      expect(find.text('February 1976'), findsOneWidget);
-      // Selected date should be painted with a colored circle
-      expect(
-          Material.of(tester.element(find.text('23'))),
-          paints..circle(color: selectedColor, style: PaintingStyle.fill)
-      );
-
-    });
-  });
-
-  group('Haptic feedback', () {
```

Comment:
Is there a reason the haptic tests were removed?

Reply:
They were moved to calendar_date_picker_test.dart.